engine/src/main/java/org/terasology/particles/updating/ParticleUpdater.java

```diff
@@ -41,18 +41,34 @@
     void dispose(EntityRef entity);
 
     /**
-     * @param emitter                    The particle emitter that is being updated.
-     * @param registeredAffectorFunctions  The list of affector functions to use when processing the given system's affectors.
+     * @param emitter The particle emitter that is being updated.
```

Comment:
Any reason you did not add a description for this method?

Reply:
Oh I missed that, those javadocs were already there from my original code last year, I didn't notice that they were incomplete. I'll go check them, thanks for pointing that out. @vampcat